src/wwwroot/controllers/BillingCtrl.js

```diff
@@ -255,6 +299,28 @@
       }
       redirectToPlanSelection();
     }
+
+    function getCustomerTypes(countryCode) {
```

Comment:
I think that getCustomerTypes should be getConsumerTypes

Reply:
Thats true! I don't understand how I got dizzy with it hahaha